packages/nvidia_nat_vanna/src/nat/plugins/vanna/db_utils.py

```
@@ -0,0 +1,298 @@
+    try:
+        with connection.connect() as conn:
+            logger.info(f"Executing query: {query}")
+            result = conn.execute(text(query))
+            rows = result.fetchall()
+            columns = list(result.keys()) if result.keys() else []
+
+            logger.info(f"Query completed, retrieved {len(rows)} rows")
+            return QueryResult(results=rows, column_names=columns)
+
+    except Exception as e:
+        logger.error(f"Error executing query: {e}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)